src/main.rs

```diff
@@ -120,16 +120,26 @@ async fn execute() -> Result<std::process::ExitCode> {
 
                     return Err(Error::Unauthorized(message))?;
                 }
-                _ => {
+                StatusCode::BAD_REQUEST => {
                     let body = response.bytes().await?;
                     let message = serde_json::from_slice::<String>(&body)?;
```

Comment:
Doesn't this still have the same issue? You got a 400 Bad Request, which had no body, so this would continue throwing an error, no?